assets/services.json

```diff
@@ -14,6 +14,7 @@
       "name": "9GAG",
       "rules": [
         "||9cache.com^",
+        "||9g.ag^",
```

Comment:
Hello @L8X
Where is this domain being used? I can't find it.

Reply:
In any case this is redirection to 9gag.com, not owned by 9gag. Please remove.